mlir/lib/Dialect/AMDGPU/Transforms/OptimizeSharedMemory.cpp

```diff
@@ -237,8 +240,10 @@ struct OptimizeSharedMemoryPass
     : public amdgpu::impl::OptimizeSharedMemoryBase<OptimizeSharedMemoryPass> {
 public:
   OptimizeSharedMemoryPass() = default;
-
+  OptimizeSharedMemoryPass(const OptimizeSharedMemoryOptions &options)
+      : OptimizeSharedMemoryBase(options) {}
```

Comment:
Nit: unless I'm missing something you can remove these 2 constructors write just: `using OptimizeSharedMemoryBase;`

Reply:
Actually, I tried that, but I am getting the error below when I am using it. Please see below:
```
llvm-project/mlir/lib/Dialect/AMDGPU/Transforms/OptimizeSharedMemory.cpp:242:9: error: using declaration requires a qualified name
  using OptimizeSharedMemoryBase;
```
If you do not mind, can I keep it as it was?